cpu/esp32/periph/gpio_ll.c

```diff
@@ -196,3 +205,19 @@ void gpio_ll_query_conf(gpio_conf_t *dest, gpio_port_t port, uint8_t pin)
     }
     irq_restore(state);
 }
+
+void gpio_ll_print_conf(const gpio_conf_t *conf)
+{
+    static const char *drive_strs[] = {
+        [GPIO_DRIVE_WEAK] = "weak",
+        [GPIO_DRIVE_STRONG] = "strong",
+    };
+    const char *drive = "invalid";
+
+    gpio_ll_print_conf_common(conf);
+    print_str(", drive: ");
+    if ((unsigned)conf->drive_strength < GPIO_DRIVE_NUMOF) {
+        drive = drive_strs[conf->drive_strength];
+    }
```

Comment:
ESP32s use four driver strengths. Since only 2 bits are used for the strength, its value is always less than `GPIO_DRIVE_NUMOF`.

```suggestion
    static const char *drive_strs[] = {
        [GPIO_DRIVE_WEAKEST] = "weakest",
        [GPIO_DRIVE_WEAK] = "weak",
        [GPIO_DRIVE_STRONG] = "strong",
        [GPIO_DRIVE_STRONGEST] = "strongest",
    };

    gpio_ll_print_conf_common(conf);
    print_str(", drive: ");
    print_str(drive_strs[conf->drive_strength]);
```